archiver/src/main.rs

```diff
@@ -1,4 +1,4 @@
-use clap::{crate_description, crate_name, crate_version, App, Arg};
+use clap::{crate_description, crate_name, App, Arg};
```

Comment:
Although I'm informed of `codemod`, I resorted to following `sed`s purely because of my laziness to learn a new tool, and managed to got the job done:

```
sed -i -e 's/crate_version, //g' $(git grep --files-with-matches crate_version, :**.rs)
sed -i -e 's/crate_version!/solana_clap_utils::version!/g' $(git grep --files-with-matches crate_version :**.rs)
```

(This is just for the record.)